prettier/src/index.ts

```diff
@@ -97,6 +99,18 @@ const tokenize = (input) => {
     });
   }
 
+  const jsonBlocks = input.match(JSON_BLOCK_REGEX);
+  if (jsonBlocks) {
+    jsonBlocks.forEach((match) => {
+      tokenIndex++;
+      tokenMap.set(
+        `<!--placeholder-${tokenIndex}-->`,
+        `{% json_block %}${match}{% end_json_block %}`
+      );
+      input = input.replace(match, `<!--placeholder-${tokenIndex}-->`);
+    });
+  }
+
```

Comment:
Just a note that I have a refactor for this logic in #49 since it's growth to be quite hairy